roles/KubevirtRepoInfo/meta/main.yml

```diff
@@ -0,0 +1,60 @@
+galaxy_info:
+  author: your name
```

Comment:
We should probably fix or remove the meta for this role.

Reply:
Also for the other roles - they are all meant to be used internally, and not published. Will fix in a new PR.